packages/vsphere/_dev/deploy/docker/Dockerfile

```
@@ -3,7 +3,7 @@ ARG VSPHERE_GOLANG_VERSION
 FROM golang:${VSPHERE_GOLANG_VERSION}-alpine
 
 RUN apk add --no-cache curl git
-RUN go get -u github.com/vmware/govmomi/vcsim@v0.25.0
+RUN go install -u github.com/vmware/govmomi/vcsim@v0.25.0
```

You might want to get rid of the `-u` flag for the install command. 
```suggestion
RUN go install github.com/vmware/govmomi/vcsim@v0.25.0
```